### ISSUE TITLE
Aria-label missing in quick links

### ISSUE THREAD

**Issue by ramya-rao-a** [Microsoft/vscode#27666]
Testing #27430 

The below links have text that are self-explanatory and so do not need aria-label

![image](https://cloud.githubusercontent.com/assets/16890566/26608856/a15331cc-4552-11e7-898a-885eb002fd2c.png)

But the links in **Recent** section list out folder names. In these links an aria-label like "Open folder" followed by the folder name followed by "with path" followed by the folder path would help screenreader users

The "More.." link in the **Recent** section should have an aria-label "Open list of recent folders"
